pandas/tests/frame/conftest.py

```diff
@@ -94,6 +94,9 @@ def mixed_float_frame():
     return df
 
 
+mixed_float_frame2 = mixed_float_frame
```

Comment:
Do you see this being used elsewhere than the one test? If not probably better just to create a copy directly within the test

Reply:
agreed don't want this here

Reply:
Think you missed removal of this in last commit